### PR TITLE
Career pages: Data Scientist, Big Data Engineer, Data Security Analyst

### PR DESCRIPTION
**Pull request by faithakinbobola**
Description, courses and skills on data science as a career path

### CODE REVIEW

**Review by Iyiola-stack — APPROVED** [review]

File: career4.html [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+        <header>
+            <div class="nav-container">
+                <div id="nav-branding">
+                    <h1>Tech<span>Hub</span>-Careers</h1>
+                </div>
+                <nav>
+                    <ul>
+                        <li><a href="index.html">Home</a></li>
+                        <li><a href="#">Careers</a></li>
+                        <li><a href="#">Wait-List</a></li>
```
Comment (goke-epapa) [inline career4.html]: Waitlist should be one word
```suggestion
                        <li><a href="#">Waitlist</a></li>
```

File: career4.html [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+                    </ul>
+                </nav>
+            </div>
+        </header>
+        <section>
+            <h2>Data Security Analyst</h2>
+            <img src="image/data-security-analyst.png" alt="data-security-analyst.png">
```
Comment (goke-epapa) [inline career4.html]: The `alt` attribute is what is shown to a user if the image doesn't load, it doesn't need to have the file extension - https://www.w3schools.com/tags/att_img_alt.asp
```suggestion
            <img src="image/data-security-analyst.png" alt="Data Security Analyst">
```

File: career4.html [hunk 3]
```diff
@@ -0,0 +1,124 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <meta charset="UTF-8">
+        <title>TechHub-Careers-data-security-analyst</title>
```
Comment (goke-epapa) [inline career4.html]:
```suggestion
        <title>TechHub-Careers - Data Security Analyst</title>
```

File: career5.html [hunk 4]
```diff
@@ -0,0 +1,95 @@
...
+        <header>
+            <div class="nav-container">
+                <div id="nav-branding">
+                    <h1>Tech<span>Hub</span>-Careers</h1>
+                </div>
+                <nav>
+                    <ul>
+                        <li><a href="index.html">Home</a></li>
+                        <li><a href="#">Careers</a></li>
+                        <li><a href="#">Wait-List</a></li>
```
Comment (goke-epapa) [inline career5.html]:
```suggestion
                        <li><a href="#">Waitlist</a></li>
```

File: career5.html [hunk 5]
```diff
@@ -0,0 +1,95 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <meta charset="UTF-8">
+        <title>TechHub-Careers-big-data-engineer</title>
```
Comment (goke-epapa) [inline career5.html]:
```suggestion
        <title>TechHub-Careers - Big Data Engineer</title>
```

File: career6.html [hunk 6]
```diff
@@ -0,0 +1,115 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <meta charset="UTF-8">
+        <title>TechHub-Careers-data-scientist</title>
```
Comment (goke-epapa) [inline career6.html]:
```suggestion
        <title>TechHub-Careers - Data Scientist</title>
```

File: career6.html [hunk 7]
```diff
@@ -0,0 +1,115 @@
...
+        <header>
+            <div class="nav-container">
+                <div id="nav-branding">
+                    <h1>Tech<span>Hub</span>-Careers</h1>
+                </div>
+                <nav>
+                    <ul>
+                        <li><a href="index.html">Home</a></li>
+                        <li><a href="#">Careers</a></li>
+                        <li><a href="#">Wait-List</a></li>
```
Comment (goke-epapa) [inline career6.html]:
```suggestion
                        <li><a href="#">Waitlist</a></li>
```

File: career6.html [hunk 8]
```diff
@@ -0,0 +1,115 @@
...
+                    </ul>
+                </nav>
+            </div>
+        </header>
+        <section>
+            <h2>Data Scientist</h2>
+            <img src="image/data-science.png" alt="data-science.png">
```
Comment (goke-epapa) [inline career6.html]:
```suggestion
            <img src="image/data-science.png" alt="Data Science">
```

File: career5.html [hunk 9]
```diff
@@ -0,0 +1,95 @@
...
+                        <li><a href="index.html">Home</a></li>
+                        <li><a href="#">Careers</a></li>
+                        <li><a href="#">Wait-List</a></li>
+                    </ul>
+                </nav>
+            </div>
+        </header>
+        <section>
+            <h2>Big Data Engineer</h2>
+            <img src="image/big-data-skills-and-dollas.png" alt="big-data-skills-and-dollas.png">
```
Comment (goke-epapa) [inline career5.html]:
```suggestion
            <img src="image/big-data-skills-and-dollas.png" alt="Big Data">
```

**Review by goke-epapa — COMMENTED** [review]
Check comments inline

**Comment by goke-epapa** [comment]
@faithakinbobola this PR should be for Data Scientist, but it has other changes for `Big Data Engineer` and `Data Security Analyst` as well

There are two options:
1. Update this PR title and close the other PR
2. Remove the unnecessary from this PR, so the title reflects what was actually done.

**Review by goke-epapa — APPROVED** [review]

**Pull request merged**